src/lib/stores/billing.ts

```diff
@@ -183,18 +183,23 @@ export function checkForTrialEnding(org: Organization) {
     }
 }
 
-export function checkForUsageLimit(org: Organization) {
+export async function checkForUsageLimit(org: Organization) {
     if (!org?.billingLimits) {
         readOnly.set(false);
         return;
     }
-    const { bandwidth, documents, executions, storage, users } = org.billingLimits;
+    const { bandwidth, documents, executions, storage, users } = org?.billingLimits;
+    const members = await sdk.forConsole.teams.listMemberships(org.$id);
+    const plan = get(plansInfo).plans.find((plan) => plan.$id === org.billingPlan);
+    const membersOverflow =
+        members?.total > plan.members ? members.total - (plan.members || Infinity) : 0;
```

Comment:
if plan.members = 0, this will make it minus infinity. Isn't it better to do:
`members?.total > plan.members ? members.total - (plan.members || members.total) : 0;`